string    name
---
manipulator_h_sdk/KinematicsPose pose
